README.md

```diff
@@ -4,79 +4,136 @@
 
 Inspired by Eric Meyers CSS Reset, Git New is a simple tool to enforce a consistent baseline Git experience for new projects.
 
-## Installation
+- [Git New](#git-new)
 
-Linux, Windows and OSX
+  - [For Project Developers](#for-project-developers)
```

Comment:
The headings / anchors here seem to be inconsistent with the content below.

Reply:
Hmm, should be auto-update on save. I will confirm.